google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -809,7 +810,12 @@ def read table, columns, keys: nil, index: nil, limit: nil,
         #   See [Data
         #   types](https://cloud.google.com/spanner/docs/data-definition-language#data_types).
         #
-        # @return [Time] The timestamp at which the operation committed.
+        # @param [Boolean] commit_stats Include statistics related to the
+        # transaction. Default it is `false`.
+        #
+        # @return [CommitResponse] It include the timestamp at which the
+        #   operation committed. It also include transaction commit stats,
+        #   if passed `commit_stats` value is `true`.
```

Comment:
I'm curious, wouldn't this be a breaking change for users? If they are currently calling `upsert`, they are expecting it to return a timestamp. But now we return a `CommitResponse` so wouldn't that throw an error?

E.g. the following code will break:
```
ts = db.upsert "users", [{ id:1, name: "Charlie", active: false }, ...]
# use ts for something else
...
```

Reply:
Updated for backward compatibility.